insert.js

```diff
@@ -0,0 +1,51 @@
+const mongoose = require('mongoose');
+const {User,Profile} = require('./model');
+let uri = 'mongodb://127.0.0.1:27017/test'
```

Comment:
make an environment  `.env`  file 
to store credentials and avoid hardcoded values in file

Reply:
ok sir i will use dotenv library